cmd/kogito/command/converter/runtimetype_converter.go

```diff
@@ -15,10 +15,16 @@
 package converter
 
 import (
-	"github.com/kiegroup/kogito-cloud-operator/api"
-	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/flag"
 	"io/ioutil"
 	"strings"
+
+	"github.com/kiegroup/kogito-cloud-operator/api"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/flag"
+)
+
+var (
+	quarkusLegacyJarSuffix = "-runner.jar"
+	quarkusRuntimeSuffixes = []string{quarkusLegacyJarSuffix, "quarkus-app", "-runner"}
```

Comment:
shouldn't it be quarkus-run or this is to identify the dir quarkus-app for fast Jar?

Reply:
renamed to `quarkusRuntimeTargetSuffixes`. Indeed this is to check in the `target` folder